### DISCUSSION TITLE
multiword should have at least one word

### DISCUSSION THREAD

**Discussion by Accacio** [cooklang/spec#31] [Q&A] [answered]
If I'm not mistaken, like it is now in the spec we can have zero length multiwords, is it right?
https://github.com/cooklang/spec/blob/9e3fdb3a7480f3ec44b6c259a3aabe30b5243938/EBNF.md?plain=1#L31

**Comment by dubadub** [comment, marked as answer]
Haha, good catch! No, that shouldn't be like that. I'll fix it.

**Reply by dubadub** [reply]
Nice, they have syntax for that which I didn't find in Wikipedia: https://stackoverflow.com/a/55643354. Updated the doc.

**Reply by Accacio** [reply]
I see you already corrected the other terms, so I'll close this :)
